packages/shared/src/components/cards/v1/RaisedLabel.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { SimpleTooltip } from '../../tooltips/SimpleTooltip';
+import classed from '../../../lib/classed';
+
+export enum RaisedLabelType {
+  Hot = 'Hot',
+  Pinned = 'Pinned',
+  Beta = 'Beta',
+}
+
+const typeToClassName: Record<RaisedLabelType, string> = {
+  [RaisedLabelType.Hot]: 'bg-theme-status-error',
+  [RaisedLabelType.Pinned]: 'bg-theme-bg-bun',
+  [RaisedLabelType.Beta]: 'bg-cabbage-40',
```

Comment:
Is this the right color, feels odd not sure it's legacy.
Should be something like `bg-theme-bg-cabbage` I feel like.

Reply:
Yes, I was playing with it a bit and `bg-theme-bg-cabbage` is the same. I guess this is just copy&paste from the old component.